### PR TITLE
[IE Python API POC]

### PR DESCRIPTION
**Pull request by akuporos**
This is POC branch where Python API is based on pybind11

Examples of using new Python API -- https://github.com/akuporos/openvino/pull/45/files

ticket #33021

### CODE REVIEW

**Comment by ilyachur** [comment]
@akuporos You really changed 1600+ files?!
I think you had some issues with git merge.

**Comment by akuporos** [comment]
> @akuporos You really changed 1600+ files?!
> I think you had some issues with git merge.

I hope @jiwaszki will fix it soon :)

**Comment by jiwaszki** [comment]
> @akuporos You really changed 1600+ files?!
> I think you had some issues with git merge.

Yeah, something went wrong there. Hope it looks good now :)

**Comment by culhatsker** [comment]
I wasn't able to `cd openvino/ngraph/python && python3 setup.py bdist_wheel` without a `ccompiler` pip package on Python 3.6.9 and Ubuntu 18.04.5 LTS

File: ngraph/python/src/pyopenvino/inference_engine/common.hpp [hunk 1]
```diff
@@ -4,15 +4,21 @@
...
 
 namespace Common {
     InferenceEngine::Layout get_layout_from_string(const std::string &layout);
 
     const std::string& get_layout_from_enum(const InferenceEngine::Layout &layout);
 
     PyObject *parse_parameter(const InferenceEngine::Parameter &param);
-};
+
+    const std::shared_ptr<InferenceEngine::Blob> convert_to_blob(const py::handle& blob);
```
[the remark this answers is not in the document]
Comment (akuporos) [reply ngraph/python/src/pyopenvino/inference_engine/common.hpp:23]: agree, fixed

**Comment by akuporos** [comment]
The same PR with hopefully totally green CI: https://github.com/openvinotoolkit/openvino/pull/7752

**Pull request closed without merging**